src/ucp/api/ucp.h

```diff
@@ -2177,8 +2177,112 @@ ucs_status_t ucp_worker_fence(ucp_worker_h worker);
  */
 ucs_status_t ucp_worker_flush(ucp_worker_h worker);
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Atomic operation requested for ucp_atomic_post
+ *
+ * This enumeration defines the type of atomic memory operation should be 
```

Comment:
s/the type of/which/
I suggest this change to make it both a bit more clear and to be consistent with the wording for the ucp_atomic_fetch enumerations.